SecurityPkg/Library/HashInstanceLibSha1/HashInstanceLibSha1.c

```diff
@@ -56,7 +56,13 @@ Sha1HashInit (
 
   CtxSize = Sha1GetContextSize ();
   Sha1Ctx = AllocatePool (CtxSize);
-  ASSERT (Sha1Ctx != NULL);
+  // MU_CHANGE [BEGIN] - CodeQL change
+  if (Sha1Ctx == NULL) {
+    ASSERT (Sha1Ctx != NULL);
+    return EFI_OUT_OF_RESOURCES;

```

Comment:
Interesting that the return value is documented but the check not actually performed.